### DISCUSSION TITLE
extension of search to comments

### DISCUSSION THREAD

**Discussion by Dom-ix** [rejetto/hfs#494] [Ideas]
It would be interesting to extend the search to comments, for a more specific search.

**Comment by rejetto** [comment]
that would be interesting for sure, although extremely slower

**Comment by rejetto** [comment]
will you give a look and tell me if it works like you need?
just go to admin-panel / home, right click on the check-for-updates and paste this link
https://github.com/user-attachments/files/18469837/hfs-windows-x64-0.56.0-alpha2.1.zip

**Reply by hugo529100** [reply]
wow!!!!! cool job it works good
